src/applet.c

```diff
@@ -227,8 +231,16 @@ static void brisk_menu_applet_init(BriskMenuApplet *self)
         gtk_widget_set_margin_end(image, 4);
         gtk_widget_set_halign(image, GTK_ALIGN_START);
 
+        /* Set the label text, if empty, show the default text */
+        gchar *text = g_settings_get_string(self->settings, "label-text");
+        if (g_str_equal(text, "")) {
+                self->label_text = _("Menu");
```

Comment:
This = bad. You have a pointer to allocated memory, and now you're using glib gettext for statically owned memory.